src/store/main.js

```diff
@@ -61,8 +61,12 @@ export default new Vuex.Store({
 		boardFilter: BOARD_FILTERS.ALL,
 		activity: [],
 		activityLoadMore: true,
+		filter: { tags: [], user: [], due: [] },
 	},
 	getters: {
+		getFilter: state => {
```

Comment:
We could just use mapState in the component that uses it so there is less code in the store.

Reply:
removed getFilter